src/control/server/storage/types.go

```diff
@@ -1,5 +1,5 @@
 //
-// (C) Copyright 2019 Intel Corporation.
+// (C) Copyright 2019-20w20 Intel Corporation.
```

Comment:
oops

Reply:
this seems to have slipped in with https://github.com/daos-stack/daos/pull/1898 , moved fix to https://github.com/daos-stack/daos/pull/1944/commits/ba17c2c83406964c03ce6c0aec59f39cd9b5cc93 which will get applied to both master and release branches.